discountService.ts

```diff
@@ -0,0 +1,13 @@
+class DiscountService {
+    getDiscount(category: string, price: number): number {
+        if (category === 'estudiante') {
+            return price * 0.2;
+        } else if (category === 'jubilado') {
+            return price * 0.15;
+        } else if (category === 'empleado') {
+            return price * 0.1;
+        } else {
+            return 0;
+        }
+    }
+}
```

Comment:
### Enhance code style
use an **object** to represent the discounts instead of multiple if's, and simplify the logic as well

Reply:
Done! Thanks for the comment.